common/src/java/com/zimbra/common/util/ZimbraCookie.java

```diff
@@ -56,11 +56,16 @@ public static boolean secureCookie(HttpServletRequest request) {
 
     public static void addHttpOnlyCookie(HttpServletResponse response, String name, String value,
             String path, Integer maxAge, boolean secure) {
-        addCookie(response, name, value, path, maxAge, true, secure);
+        addCookie(response, name, value, path, maxAge, true, secure, false);
```

Comment:
here we should call overloaded method addHttpOnlyCookie() with default value false for new param instead of addCookie().
addCookie will get called by overloaded method addHttpOnlyCookie()